data/eventsData.js

```diff
@@ -0,0 +1,30 @@
+import { chapters, initiatives } from '../src/constants';
+
+const events = [
```

Comment:
Za mě bych strukturu eventů udělal jako jeden dokument. Ano mohli bychom to oddělit, ale jestli detail je jen o tři/čtyři parametry delší, tak je to zbytečné, to jsou jen setinky dat. 

Přidal jsem tam ids abychom to později mohli vázat i na strukturu chapterů a organizátorů + reglink

Jo a nevím co je stag :D

Jinak good job, i to původní by fungovalo, jen zbytečně komplikované

```
{
	chapterId: "chapters/bladad", /// Bude abychom popr udrzovali events updated,
	chapterName: "GDG Jihlava",
        eventName: "Name",
        date: '',
        address: '',
        stag: 'name',
        intro: 'Dlouhý text',
        location: '',
        image: 'path/to/image',
	regLink: "sfdsf"
	intro: "Dlouhý text",
        mdText: 'Hodně dlouhý text v **markdownu**',
        organizers: [{ id: '', image: '', name: '', stag: '' }],    
}
```

Reply:
@ryzizub dík moc!

ok. beru. Když to bude problém, tak to přepracujeme. 

Stag měl být slug ale nemohl jsem si v daný moment vzpomenout :D
https://en.wikipedia.org/wiki/Slug_(publishing)